SPEC.md

```diff
@@ -447,10 +447,10 @@ When evaluating `ac-discovery` tags, the client MUST first ensure that the prefi
 
 ```bash
 urltmpl="https://{name}-{version}-{os}-{arch}.{ext}"
-curl $(echo "$urltmpl" | sed -e "s/{name}/$appname/" -e "s/{version}/$version/" -e "s/{os}/$os/" -e "s/{arch}/$arch/" -e "s/{ext}/$ext/")
+curl $(echo "$urltmpl" | sed -e "s/{name}/$imgname/" -e "s/{version}/$version/" -e "s/{os}/$os/" -e "s/{arch}/$arch/" -e "s/{ext}/$ext/")
```

Comment:
This should just be name, I think, not sure why it was appname in the first place?